### PR TITLE
VR-180 Extended backpack design!

### PR DESCRIPTION
**Pull request by SolarCrane**

### CODE REVIEW

File: CoreScriptsRoot/Modules/BackpackScript.lua [hunk 1]
```diff
@@ -1507,7 +1576,84 @@ local function OnVREnabled(prop)
...
+			CloseInventoryButton.ImageTransparency = BackpackPanel.transparency
+			LeftBumperButton.ImageTransparency = BackpackPanel.transparency
+			RightBumperButton.ImageTransparency = BackpackPanel.transparency
+		end
+
+		OpenInventoryButton.Parent = MainFrame
+		CloseInventoryButton.Parent = InventoryFrame
+		-- LeftBumperButton.Parent = HotbarFrame
+		-- RightBumperButton.Parent = HotbarFrame
+
```
Comment (jmargh) [inline CoreScriptsRoot/Modules/BackpackScript.lua]: LeftBumperButton and RightBumperButton are not parented. Are these going to be used in the future?

File: CoreScriptsRoot/Modules/BackpackScript.lua [hunk 2]
```diff
@@ -1142,6 +1168,49 @@ for i = 1, HOTBAR_SLOTS do
...
+	CloseInventoryButtonSelectionObject.Size = UDim2.new(0.9,0,0.9,0);
+	CloseInventoryButtonSelectionObject.Position = UDim2.new(0.05,0,0.05,0);
+	CloseInventoryButtonSelectionObject.Image = "rbxasset://textures/ui/Keyboard/close_button_selection.png";
+	CloseInventoryButton.SelectionImageObject = CloseInventoryButtonSelectionObject
+
+	CloseInventoryButtonIcon = NewGui('ImageLabel', 'Icon')
+	CloseInventoryButtonIcon.Size = UDim2.new(0.5,0,0.5,0);
+	CloseInventoryButtonIcon.Position = UDim2.new(0.25,0,0.25,0);
+	CloseInventoryButtonIcon.Image = "rbxasset://textures/ui/Keyboard/close_button_icon.png";
+	CloseInventoryButtonIcon.Parent = CloseInventoryButton;
```
Comment (jmargh) [inline CoreScriptsRoot/Modules/BackpackScript.lua]: Are both CloseInventoryButtonSelectionObject and CloseInventoryButtonIcon intended to be global?

File: CoreScriptsRoot/Modules/BackpackScript.lua (hunk 1 above)
Comment (SolarCrane) [inline CoreScriptsRoot/Modules/BackpackScript.lua]: Yes, but we don't have art for it yet

File: CoreScriptsRoot/Modules/BackpackScript.lua (hunk 2 above)
Comment (SolarCrane) [inline CoreScriptsRoot/Modules/BackpackScript.lua]: They shouldn't be since I put them in the do end block here

**Pull request merged**